src/Resources/config/amqp.xml

```diff
@@ -5,11 +5,13 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="wisembly.amqp.publisher" class="Wisembly\AmqpBundle\Publisher">
-            <argument type="service" id="event_dispatcher" />
-            <argument type="service" id="wisembly.amqp.broker" /> <!-- declared in a compiler pass //-->
-        </service>
+        <defaults public="false" autowire="true" autoconfigure="true" />
 
-        <service id="wisembly.amqp.gates" class="Wisembly\AmqpBundle\GatesBag" />
+        <service id="Wisembly\AmqpBundle\Publisher" public="true" />
+        <service id="Wisembly\AmqpBundle\GatesBag" />
```

Comment:
why this service is private and its alias public ?

Reply:
backward compatibility